resource-timing/resources/iframe-TAO-crossorigin-port.sub.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script>
+    const url = '{{location[scheme]}}://{{host}}:{{ports[http][1]}}/resource-timing/resources/TAOResponse.py?tao=origin_port_{{ports[http][1]}}';
+    let observe = (list, observer) => {
+        let entry = list.getEntries()[0];
+        if (!entry) {
+            window.top.postMessage("FAIL - no entry", '*');
+            return;
+        }
+        let sum = entry.redirectStart +
+                  entry.redirectEnd +
+                  entry.domainLookupStart +
+                  entry.domainLookupEnd +
+                  entry.connectStart +
+                  entry.connectEnd +
+                  entry.secureConnectionStart +
+                  entry.requestStart +
+                  entry.responseStart +
+                  entry.transferSize +
+                  entry.encodedBodySize +
+                  entry.decodedBodySize;
+
+        let result = sum == 0 ? 'PASS' : 'FAIL';
```

Comment:
Isn't this saying pass when the check failed?

Reply:
Yeah, we expect TAO not to pass, as the ports don't line up